tensorflow/contrib/rnn/BUILD

```diff
@@ -26,7 +26,8 @@ load(
 
 tf_custom_op_py_library(
     name = "rnn_py",
-    srcs = ["__init__.py"] + glob(["python/ops/*.py"]),
+    srcs = ["__init__.py"] + glob(["python/ops/*.py"]) 
+                           + glob(["python/tools/*.py"]),
```

Comment:
Works for me. I actually did it that way at first but changed my mind. I've updated this PR.